base/abstractarray.jl

```diff
@@ -803,9 +825,8 @@ typed_hcat(T::Type, A::AbstractArray...) = cat_t(2, T, A...)
 function hvcat(nbc::Integer, as...)
     # nbc = # of block columns
     n = length(as)
-    if mod(n,nbc) != 0
-        throw(ArgumentError("all rows must have the same number of block columns"))
-    end
+    mod(n,nbc) != 0 &&
+        throw(ArgumentError("all rows must have the same number of block columns ($nbc)"))
```

Comment:
Here, I'm not sure how to do that.  This function seems to check just the length of the tuple in `as` using `mod`, I'm not even sure what gets passed in `as` (I'll add some comments and/or descriptive variable names if you tell me what it means!)